server/db.js

```diff
@@ -0,0 +1,14 @@
+var OrientDB = require('orientjs');
+
+var server = OrientDB({
+  host: process.env.CONNECTION_URL,
+  port: 2424,
+  username: process.env.USER,
+  password: process.env.PASSWORD
```

Comment:
Please change these env variable names to: `DB_HOST`, `DB_USER`, and `DB_PASSWORD`